### PR TITLE
Update service creation for Promscale steps

### PR DESCRIPTION
**Pull request by VineethReddy02**
# Description

Update one of the Promscale service creation steps to include base resource requirements details. 

## Subject matter expert (SME) review checklist

- [ ] Is the content technically accurate?
- [ ] Is the content complete?
- [ ] Is the content presented in a logical order?
- [ ] Does the content use appropriate names for features and products?
- [ ] Does the content provide relevant links to further information?

## Documentation team review checklist

- [ ] Is the content free from typos?
- [ ] Does the content use plain English?
- [ ] Does the content contain clear sections for concepts, tasks, and references?
- [ ] Have any images been uploaded to the correct location, and are resolvable?
- [ ] If the page index was updated, are redirects required
      and have they been implemented?
- [ ] Have you checked the built version of this content?

### CODE REVIEW

File: _partials/_cloud-create-service-promscale.md [hunk 1]
```diff
@@ -7,12 +7,16 @@ Each service contains a single database, named `tsdb`.
...
+1.  Click `Advanced configuration` and set the `Region`, `Compute`, and `Disk size`
+    for the service. With 1&nbsp;CPU, 4&nbsp;GB and 250&nbsp;GB you can ingest and
+    store up to 10,000&nbsp;metric samples per second with the default 3-month retention. 
+    Check our [resource recommendation guide][promscale-resource-recomm] for configuring the compute and 
+    disk size based on your ingestion rate and retention.
+    <highlight type="note">
+    Make sure the Promscale Connector and the service in Timescale cloud are
+    on the same region. Ideally we recommend the latency between the Promscale 
+    Connector and Database to be less than 100&nbsp;ms. 
+    </highlight>
```
Comment (Loquacity) [inline _partials/_cloud-create-service-promscale.md:19]:
```suggestion

    <highlight type="note">
    Make sure the Promscale Connector and the service in Timescale Cloud are
    in the same region. Ideally, the latency between the Promscale 
    Connector and database should be less than 100&nbsp;ms. 
    </highlight>
    
```

**Comment by Loquacity** [comment]
@VineethReddy02 Do you still want to merge this?

**Comment by VineethReddy02** [comment]
Yes, totally forgot about this PR.

File: _partials/_cloud-create-service-promscale.md [hunk 2]
```diff
@@ -7,12 +7,18 @@ Each service contains a single database, named `tsdb`.
...
-1.  Click  `Advanced configuration` and set the `Region`, `Compute`, and `Disk size`
-    for the service.
-    To store your observability data it is recommended that you allocate
-    at least 4&nbsp;CPU, 16&nbsp;GB of memory, and 50&nbsp;GB of disk equivalent to
-    840&nbsp;GB of uncompressed data. You can scale up this configuration later if
-    your data ingestion and query rate increases.
+1.  Click `Advanced configuration` and set the `Region`, `Compute`, and `Disk size`
+    for the service. With 1&nbsp;CPU, 4&nbsp;GB and 250&nbsp;GB you can ingest and
+    store up to 10,000&nbsp;metric samples per second with the default 3-month retention. 
+    Check our [resource recommendation guide][promscale-resource-recomm] for configuring the compute and
```
Comment (Loquacity) [inline _partials/_cloud-create-service-promscale.md:13]:
```suggestion
    Check the [resource recommendation guide][promscale-resource-recomm] for configuring the compute and 
```

**Review by Loquacity — APPROVED** [review]

**Pull request merged**